package.yaml

```diff
@@ -25,10 +25,11 @@ dependencies:
 library:
   source-dirs:      src
   dependencies:
-    - ghc >= 8.10.2 && < 9.0
+    - ghc >= 8.10.2 && <9.4
     - containers >= 0.6.2.1
     - term-rewriting >= 0.3.0.1
     - transformers >= 0.5.6.2
+  default-extensions:  CPP
```

Comment:
I list the language extensions in every file because some tools (e.g. formatters which transform stdin to stdout) can't look at the cabal file, they can only look at the current file.

Reply:
Fixed in #37